### ISSUE TITLE
A little question about ratings_diag in BCLoss_Batch

### ISSUE THREAD

**Issue by LeavesLi1015** [anzhang314/BC-Loss#2]
Hello,
I'm currently working on graph contrastive learning and I really appreciate your work! However, I get a bit confused by the `ratings_diag` in BCLoss function. It's `ratings_diag = torch.cos(torch.arccos(torch.clamp(ratings_diag,-1+1e-7,1-1e-7))+(1-torch.sigmoid(pos_ratings_margin)))` and `pos_ratings = torch.cos(torch.arccos(torch.clamp(pos_ratings,-1+1e-7,1-1e-7))+(1-torch.sigmoid(pos_ratings_margin)))` that confuses me. I can't associate M_{ui} in the equation(6) with `1-sigmoid(pos_ratings_margin)`. Could you provide an explaination? Thanks a lot!

Best,
Lucas

![image](https://user-images.githubusercontent.com/76167689/222470214-f796ade2-7725-4842-a3d7-df86bb2cca39.png)
